test/lib/usd/utils/CMakeLists.txt

```diff
@@ -33,3 +33,62 @@ mayaUsd_add_test(${TARGET_NAME}
     ENV
         "LD_LIBRARY_PATH=${ADDITIONAL_LD_LIBRARY_PATH}"
 )
+
+set(TARGET_NAME MayaTransformAPI)
+
+add_executable(${TARGET_NAME})
+
+# -----------------------------------------------------------------------------
+# sources
+# -----------------------------------------------------------------------------
+target_sources(${TARGET_NAME}
+    PRIVATE
+        main.cpp
+        test_MayaTransformAPI.cpp
+)
+add_definitions(
+  -DUSDUTILS_TEST_DATA="${CMAKE_CURRENT_SOURCE_DIR}/test_data"
+)
```

Comment:
don't use `add_definitions`, instead use  `target_compile_definitions`

```
# -----------------------------------------------------------------------------
# compile configuration
# -----------------------------------------------------------------------------
target_compile_definitions(${TARGET_NAME}
    PRIVATE
        USDUTILS_TEST_DATA="${CMAKE_CURRENT_SOURCE_DIR}/test_data"
)
```
Please see the cmake section in the guidelines:
https://github.com/Autodesk/maya-usd/blob/dev/doc/codingGuidelines.md